src/style.css

```
@@ -1,3 +1,75 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+.logo img {
+  max-height: 40px;
+  margin-left: 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  font-weight: 500;
+  min-width: 1024px;
+  position: relative;
```

Don't need to use position here
```suggestion
  width: 100%;
```